app/src/sharedTest/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentTest.kt

```diff
@@ -1311,24 +1270,62 @@ class PlatformParametersFragmentTest {
         atPositionOnView(
           recyclerViewId = R.id.platform_parameters_recycler_view,
           position = 0,
+          targetViewId = R.id.reset_button
+        )
+      ).perform(click())
+
+      pressBack()
+      testCoroutineDispatchers.runCurrent()
+
+      onView(withText(R.string.pending_changes_dialog_title_text))
+        .inRoot(isDialog())
+        .check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testPlatformParametersFragment_toggleParamOnAndOff_naviagteback_skipsPendingChangesDialog() {
```

Comment:
```suggestion
  fun testPlatformParametersFragment_toggleParamOnAndOff_navigateBack_skipsPendingChangesDialog() {
```
Also, not sure about naming parameters as toggle, since parameters can be text values as well, would prefer something like: 
```
fun testPlatformParametersFragment_revertParamChange_navigateBack_skipsPendingChangesDialog() {}
```  
but feel free proceed with any if it was suggested to be followed on `codeowner` review, as it's not a major blocker.

Reply:
Updated

Reply:
Updated

Reply:
Latest naming looks better than my original suggestion.